### PR TITLE
Disallow sqlite3 2.x

### PR DESCRIPTION
**Pull request by tagliala**
Ref: rails/rails#51636

---

Not quite sure of this fix, especially on the dependabot side

### CODE REVIEW

**Comment by javierjulio** [comment]
@tagliala this would be right as I had to [lock the version to v1 for the bug report template scheduled workflow](https://github.com/activeadmin/activeadmin/commit/125688645f761f6ea4df5fa2d0868caa7318d108) for that test to pass. For the Dependabot config file change, wouldn't we want to duplicate that for the other bundler entries?

**Comment by javierjulio** [comment]
Ok great, thank you!

File: gemfiles/rails_70/Gemfile.lock [hunk 1]
```diff
@@ -371,6 +377,7 @@ GEM
 PLATFORMS
   arm64-darwin-21
   arm64-darwin-22
+  ruby
```
Comment (tagliala) [inline gemfiles/rails_70/Gemfile.lock:380]: this is being added by my local configuration. I have a global `force_ruby_platform` setting

**Comment by tagliala** [comment]
Supersedes #8315, #8316, #8320

**Review by javierjulio — APPROVED** [review]
Thanks!

**Pull request merged**